website/docs/blog.mdx

```diff
@@ -568,7 +568,22 @@ https://example.com/blog/feed.json
 
 ### Blog-only mode {#blog-only-mode}
 
-You can run your Docusaurus 2 site without a dedicated landing page and instead have your blog's post list page as the index page. Set the `routeBasePath` to be `'/'` to serve the blog through the root route `example.com/` instead of the subroute `example.com/blog/`.
+You can run your Docusaurus 2 site without a dedicated landing page and instead have your blog's post list page as the index page. Set the `routeBasePath` to be `'/'` to serve the blog through the root route `example.com/` instead of the subroute `example.com/blog/`. Ensure that all links pointing to `/blog` are changed to `/` as well.
+
+:::caution
+
+You should only set the `routeBasePath` to be `'/'` if you do not want a homepage. It is possible to remove the docs functionality while still having a separate homepage.
+
+If you do chose to forego a landing page by setting `routeBasePath` to `'/'`, don't forget to delete the existing homepage at `./src/pages/index.js` or else there will be two files mapping to the same route! If this is the case, you will see a warning in your terminal that says something like:
```

Comment:
we don't have such feature currently but it probably makes sense to let users configure the blog homepage path (ie the first page of the pagination)

We already do this for a few pages like the archive page: https://docusaurus.io/docs/next/api/plugins/@docusaurus/plugin-content-blog#archiveBasePath

`routeBasePath: '/'` + `blogHomePath: "/home"` could make sense and not conflict with your `pages/index.js` route 🤷‍♂️